### PR TITLE
Add random-color extension

### PR DESCRIPTION
**Pull request by srcrip**
## Description

This is a new command for making random colors.

## Screencast

CleanShot 2024-01-06 at 08 45 44@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by sevensidedmarble** [comment]
@pernielsentikaer I think mine is different. None of the others are really focused on random color generation. I would really like to get it merged! but I don't understand the failure here. It says my github name is wrong or something, but that is my name on github.

**Comment by sevensidedmarble** [comment]
To expand on whats different here, most of the alternatives are one of the following:
- wrappers around an external command that I (and others may not) don't use
- trying to be a dropper, which this is not interested in
- finally the most popular one is built around saving colors you want for later. this is for a color you want to throw away, when you just need a random thing for something you're designing.

**Comment by pernielsentikaer** [comment]
What's your [Raycast handle](https://www.raycast.com/pernielsentikaer) - it seems like it's not right 🙂

**Comment by sevensidedmarble** [comment]
ok fixed: https://www.raycast.com/sevensidedmarble

Can you rerun the checks? I don't seem to have the permission to do so without pushing a new commit

File: extensions/random-color/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "categories": [
+    "Design Tools",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "random-color",
+      "title": "Random Color",
+      "subtitle": "Pick a random color.",
```
Comment (pernielsentikaer) [inline extensions/random-color/package.json:17]:
```suggestion
```

File: extensions/random-color/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  "author": "sevensidedmarble",
+  "categories": [
+    "Design Tools",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "random-color",
+      "title": "Random Color",
```
Comment (pernielsentikaer) [inline extensions/random-color/package.json:16]:
```suggestion
      "title": "Copy Random Color",
```

File: extensions/random-color/package.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      "subtitle": "Pick a random color.",
+      "description": "Get a random color, copy it to your clipboard, or paste it into the current app.",
+      "mode": "no-view"
+    },
+    {
+      "name": "select-color",
+      "title": "Select Color",
+      "subtitle": "Choose from some random colors.",
```
Comment (pernielsentikaer) [inline extensions/random-color/package.json:24]:
```suggestion
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- You're missing an extension icon, you can generate [one here](https://icon.ray.so/) 

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
I fixed a bug since no preference values was called `clipboard` - could you check that everything is still working 🙂

**Comment by srcrip** [comment]
Since I'm using the raycast CLI, how exactly do you pull remote changes from this repo?

**Comment by pernielsentikaer** [comment]
You should be able to pull them with `npx @raycast/api@latest pull-contributions`

**Comment by srcrip** [comment]
Ok I fixed a little issue, you should be able to re-review now.

When I want to release a new version, does it go through the same PR process btw?

**Comment by pernielsentikaer** [comment]
Hi @srcrip 👋

Yes, it will, but it should be much faster since PRs with `OP is Author` are prioritized 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**